openpype/pipeline/colorspace.py

```diff
@@ -13,12 +13,17 @@
     Logger
 )
 from openpype.pipeline import Anatomy
+from openpype.lib.transcoding import VIDEO_EXTENSIONS, IMAGE_EXTENSIONS
+
 
 log = Logger.get_logger(__name__)
 
 
-class CashedData:
+class CachedData:
     remapping = None
```

Comment:
```suggestion
    remapping = {}
```

Where are we setting the `remapping` attribute to `dict`?
We can do `CachedData.remapping.setdefault`, which would then be: `None.setdefault(key, {})`

Reply:
I'm pretty sure e.g. `get_remapped_colorspace_to_native` won't work currently.

Reply:
Great catch, thanks!